lessons/java/07-Javaのメソッド問題/00-コースに含まれる単語.md

```diff
@@ -0,0 +1,53 @@
+# コースに含まれる単語
+
+| ひらがな/カタカナ | 漢字 | 英語 |
+| --------- | --------- | ------------------- |
+| しゅような | 主要な | Major、Important |
+| ていぎする | 定義する | Define |
+| きほん | 基本 | Basic |
+| かいせつする | 解説する | Explain |
+| インスタンス | - | Instance |
+| クラス | - | Class |
+| きじゅつする | 記述する | Describe |
+| じっこうする | 実行する | Execute |
+| すでに | 既に | Already |
+| アクセス | - | Access |
+| しゅうしょくし | 修飾子 | Modifier |
+| フィールド | - | Field |
+| かのう | 可能 | Possible |
+| もどりち | 戻り値 | Return Value  |
+| データせい | データ型 | Data Type |
+| していする | 指定する | Specify |
+| しきべつし | 識別子 | Identifier |
+| ひきすう | 引数 | Argument |
+| かくのうする | 格納する | Store |
+| かっこ | 括弧 | Bracket |
+| ふくすうこ | 複数個 | Multiple |
+| くぎる | 区切る | Divide |
+| ブロック | - | Block |
+| しゅるい | 種類 | Type |
+| わたす | 渡す | Hand over |
+| にんい | 任意 | Any |
+| いち | 位置 | Position |
+| ぶんき | 分岐 | Branch |
+| いずれか | 何か | Either |
+| はなれる | 離れる | Separated |
```

Comment:
こちらです。
https://github.com/reytech-co-jp/yume-project/blob/39a05939708c6aaae9425cf0a80e9624ee5f5996/lessons/java/07-Java%E3%81%AE%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89%E5%95%8F%E9%A1%8C/01-Java%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89%E3%81%AE%E5%8B%89%E5%BC%B7%E8%A8%98%E4%BA%8B.md?plain=1#L320

Reply:
ありがとうございます。
この場合、英語にするとseparateにはならないと思います。
> It is convenient to use a method when the code is written in a distant location within the program.

> If the code is written far away in the program, it can be useful to use a method.

Google翻訳やChatGPTを使うとよいです。

Reply:
「離れる」の代わりに「離れたところ」と書いて distant location と書いたらよろしいでしょうか？

Reply:
そうですね。
でもdistance locationだとしてもすこし違和感がありますね……
ここは文章で紹介するほうがいいかもしれませんね。

Reply:
いいと思います！